notifier.rst

```diff
@@ -912,3 +912,32 @@ is dispatched. Listeners receive a
+.. _`FakeSms`: https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Notifier/Bridge/FakeSms/README.md
+.. _`FreeMobile`: https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Notifier/Bridge/FreeMobile/README.md
+.. _`GatewayApi`: https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Notifier/Bridge/GatewayApi/README.md
+.. _`Infobip`: https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Notifier/Bridge/Infobip/README.md
+.. _`Iqsms`: https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Notifier/Bridge/Iqsms/README.md
+.. _`LightSms`: https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Notifier/Bridge/LightSms/README.md
+.. _`Mailjet`: https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Notifier/Bridge/Mailjet/README.md
+.. _`MessageBird`: https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Notifier/Bridge/MessageBird/README.md
+.. _`MessageMedia`: https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Notifier/Bridge/MessageMedia/README.md
+.. _`Mobyt`: https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Notifier/Bridge/Mobyt/README.md
+.. _`Nexmo`: https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Notifier/Bridge/Nexmo/README.md
+.. _`Octopush`: https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Notifier/Bridge/Octopush/README.md
+.. _`OvhCloud`: https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Notifier/Bridge/OvhCloud/README.md
+.. _`Sendinblue`: https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Notifier/Bridge/Sendinblue/README.md
+.. _`Sms77`: https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Notifier/Bridge/Sms77/README.md
+.. _`Sinch`: https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Notifier/Bridge/Sinch/README.md
+.. _`Smsapi`: https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Notifier/Bridge/Smsapi/README.md
+.. _`SmsBiuras`: https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Notifier/Bridge/SmsBiuras/README.md
+.. _`Smsc`: https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Notifier/Bridge/Smsc/README.md
+.. _`SpotHit`: https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Notifier/Bridge/SpotHit/README.md
+.. _`Telnyx`: https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Notifier/Bridge/Telnyx/README.md
+.. _`TurboSms`: https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Notifier/Bridge/TurboSms/README.md
+.. _`Twilio`: https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Notifier/Bridge/Twilio/README.md
+.. _`Vonage`: https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Notifier/Bridge/Vonage/README.md
+.. _`Yunpian`: https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Notifier/Bridge/Yunpian/README.md
```

Comment:
I don't like this way (5.4 in URL and absolute url), but I don't have solution.

Maybe we can have a new keyword like `:class`,`:method:` in order to resolve theses on DocsBuilder ?

Reply:
ping @weaverryan @nikophil

Reply:
Hello!

those already exist.
for instance here: https://symfony.com/doc/current/console.html#output-sections the two links `ConsoleOutput::section()` and `ConsoleSectionOutput` are generated with respectively:
```rst
:method:`ConsoleOutput::section() <Symfony\\Component\\Console\\Output\\ConsoleOutput::section>`
```
and
```rst
:class:`Symfony\\Component\\Console\\Output\\ConsoleSectionOutput`
```
(you can find the source here: https://raw.githubusercontent.com/symfony/symfony-docs/6.2/console.rst)

Reply:
Oops, I knew these exist, but is it possible t generate a link to markdown file to symfony repo ?

Reply:
ok sorry, I understand now. Maybe what's needed is a new `:file:` ? :thinking:

Reply:
I created a PR on docs-builder https://github.com/symfony-tools/docs-builder/pull/149 on updated this PR in order to use `:file:` and confirm it can works